index.css

```diff
@@ -0,0 +1,45 @@
+body{
+    background-color: white;
+    font-size: 16px;
+    color: #241c15;
+    font-family: Means Web,Georgia,Times,Times New Roman,serif;
+    line-height: 1.35;
+    overflow-x: hidden;
+    text-align: center;
+    margin: 0px;
+}
+
+
+.navbar{
+    width: 100%;
+    height: 50px;
+    margin-top: 0px;
+    margin-right: 20px;
+    padding: 0px;
+    
+}
+.one{
+    text-align: right;
+    background-color: yellow;
+    width: 100%;
+    top: 0px;
+    left: 0px;
+    padding: 10px;
+    border: none;
+    margin-right: 30px;
+    padding-right: 20px;
+   
+   
+}
+button {
+    margin-right: 20px;
+}
+a{
+    padding: 5px;
+    padding-top: 10px;
+}
+.flex4{
```

Comment:
Wrong display of flex.  The flex display and the direction are set in the parent container while the portion is displayed in each child element if they are occupying different heights and widths.

Reply:
Please make sure to pass your comments in a way that can be easily understood, the above issue could have started with a screen shot of the UI, then explain what is expected and why

Reply:
- [Optional] I think it is better to use a couple of emojis and/or gifs in the review to make the code review more human-friendly 👽🤖.